calico-enterprise_versioned_docs/version-3.17/getting-started/compatibility.mdx

```diff
@@ -37,79 +37,46 @@ Note that all Windows feature limitations are described in [Windows limitations]
 
 Kubernetes version support aligns with [upstream Kubernetes](#kubernetes-kubeadm) to the latest version if available.
 
-| {{prodname}} version | {{prodname}} support                                                                     |
-| -------------------- | ---------------------------------------------------------------------------------------- |
-| 3.16                 | - {{prodname}} CNI with network policy<br />- Azure CNI with {{prodname}} network policy |
-| 3.15                 | - {{prodname}} CNI with network policy<br />- Azure CNI with {{prodname}} network policy |
-| 3.14                 | - {{prodname}} CNI with network policy<br />- Azure CNI with {{prodname}} network policy |
-| 3.13                 | Azure CNI with {{prodname}} network policy                                               |
-| 3.12                 | Azure CNI with {{prodname}} network policy                                               |
-| 3.11                 | Azure CNI with {{prodname}} network policy                                               |
-| 3.10                 | Azure CNI with {{prodname}} network policy                                               |
-| 3.9                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.8                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.7                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.6                  | Azure CNI with {{prodname}} network policy                                               |
-| 3.5                  | Azure CNI with {{prodname}} network policy                                               |
+| {{prodname}} version | {{prodname}} support                                         |
+| -------------------- | ------------------------------------------------------------ |
+| 3.14 through current release         | - {{prodname}} CNI with network policy<br />- Azure CNI with {{prodname}} network policy |
+| 3.25 to 3.13         | Azure CNI with {{prodname}} network policy                   |
```

Comment:
Just wondering if this should be 3.13 to 3.25.

Also, I do not see the version 3.17 in the deploy preview.